Test/dafny0/ResolutionErrors.dfy.expect

```diff
@@ -75,6 +75,8 @@ ResolutionErrors.dfy(491,8): Error: when allocating an object of type 'Luci', on
 ResolutionErrors.dfy(493,15): Error: class Lamb does not have an anonymous constructor
 ResolutionErrors.dfy(546,7): Error: RHS (of type List<A>) not assignable to LHS (of type List<B>) (covariant type parameter would require A <: B)
 ResolutionErrors.dfy(551,7): Error: RHS (of type List<A>) not assignable to LHS (of type List<B>) (covariant type parameter would require A <: B)
+ResolutionErrors.dfy(551,7): Error: RHS (of type List<A>) not assignable to LHS (of type List<B>) (covar
```

Comment:
Why do we have this error in double now?

Reply:
mea culpa; I reverted the non-changing unit test changes by hand and I must have made a typo during so.  (I wonder why running tests locally didn't catch this?  Something to investigate on my end.)